vcls/pushmode/compose.yaml

```diff
@@ -0,0 +1,68 @@
+services:
+  lb:
+    depends_on:
+      - push-origin
+    image: 'quay.io/varnish-software/varnish-plus:latest'
+    volumes:
+      - ./load-balancer.vcl:/etc/varnish/default.vcl
+    ports:
+      - "6081:6081"
+
+  push-origin: &varnish
+    image: 'quay.io/varnish-software/varnish-plus:latest'
+    deploy:
+      replicas: 3
+    volumes:
+      - ./push.vcl:/etc/varnish/default.vcl
+
+  broadcaster:
+    image: 'quay.io/varnish-software/varnish-broadcaster:latest'
+    restart: always
+    environment:
+      VARNISH_BROADCASTER_CFG: /tmp/nodes/nodes.conf
+      VARNISH_BROADCASTER_EXTRA: -confwatch 1s
+    volumes:
+      - node_dir:/tmp/nodes
+    depends_on:
+      - discovery
+
+  discovery:
+    image: 'quay.io/varnish-software/varnish-discovery:latest'
+    environment:
+      VARNISH_DISCOVERY_FLAGS: dns --group push-origin --nodefile /tmp/nodes/nodes.conf --port 6081
+    volumes:
+      - node_dir:/tmp/nodes
+
+  live_video_generator:
+    image: 'linuxserver/ffmpeg:latest'
+    depends_on:
+      - lb
+    volumes:
+      - ./background.jpg:/srv/background.jpg
+    command: >
+      -threads 4 -re -loop 1
+      -i /srv/background.jpg
+      -vf drawtext="fontfile=monofonto.ttf: fontsize=84: box=0: fontcolor=white: x=775: y=890: text='%{gmtime\\:%H\\\\\\:%M\\\\\\:%S}',
```

Comment:
can you simplify that using https://github.com/varnish/toolbox/blob/main/docker-compose-examples/video-streaming/build_live_hls.sh#L14 or similar? I'd rather keep it straightforward and I don't want to keep a binary file around if we can help it

Reply:
Yeah, less is more and ffmpeg includes a bunch of test generators already